.github/workflows/build.yml

```diff
@@ -20,9 +20,7 @@ jobs:
           go-version: ^1.18
 
       - name: Set up Rust
-        run: |
-          rustup set profile minimal
-          rustup component add rustfmt clippy
+        run: rustup show
```

Comment:
You explained it in the PR description, however, it would be awesome to add a comment here that explains this installs rust using rust-toolchain.toml.

We are still bunch of newbies in the rust ecosystem :D